.azure-pipelines/pipelines.yml

```diff
@@ -23,8 +23,23 @@ variables:
 - name: isStableBranch
   # A release branch can be either `main` or a `release/v1.x` stable branch
   value: $[or(eq(variables['isMain'], 'true'), eq(variables['isReleaseBranch'], 'true'))]
+- name: nonMobileFilesChanged
+  value: True
 
 stages:
+- stage: change_detection
+  jobs:
+  - job: repo_change
+    dependsOn: []
+    pool:
+      vmImage: "ubuntu-20.04"
+    steps:
+    - bash: |
+        git diff --name-only origin/main | grep -qe common/ \
```

Comment:
thinking about this we could skip coverage for a whole bunch of changes

Reply:
a much better (working) suggestion for the git command to the one above:

```bash
git diff-index --quiet upstream/$(System.PullRequest.TargetBranch) :^mobile \
    || ...
```

Reply:
this is still problematic as it only works on pull requests

for main/release branches we probably want to compare to the last commit (or commit of last landed pr in case of release branches)